### DISCUSSION TITLE
Including blog images return "nil pointer evaluating resource.Resource.Resize"

### DISCUSSION THREAD

**Discussion by iamjatindersingh** [thuliteio/doks#410] [Q&A]
I am unable to include images in blog articles. It returns:

execute of template failed: template: shortcodes/img.html:2:18: executing "shortcodes/img.html" at <$image.Resize>: nil pointer evaluating resource.Resource.Resize

Any pointers are appreciated. Thanks!

**Comment by h-enk** [comment]
See #186
